docs/examples/Classifying_Using_HMM.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+Classification Using Hidden Markov Model
+========================================
+This is a demonstration using the implemented Hidden Markov model to classify multiple targets.
+
+We will attempt to classify 3 targets in an undefined region.
+Our sensor will be all-seeing, and provide us with indirect observations of the targets such that,
+using the implemented hidden Markov Model (HMM), we should hopefully successfully classify exactly
+3 targets correctly.
+"""
+
+# %%
+# All Stone Soup imports will be given in order of usage.
+
+from datetime import datetime, timedelta
+import numpy as np
+
+# %%
+# Ground Truth
+# ^^^^^^^^^^^^
+# The targets may take one of three discrete hidden classes: 'bike', 'car' and 'bus'.
+# It will be assumed that the targets cannot transition from one class to another, hence an
+# identity transition matrix is given to the :class:`~.CategoricalTransitionModel`.
+#
+# A :class:`~.CategoricalState` class is used to store information on the classification/category
+# of the targets. The state vector will define a categorical distribution over the 3 possible
+# classes, whereby each component defines the probability that a target is of the corresponding
+# class. For example, the state vector (0.2, 0.3, 0.5), with category names ('bike', 'car', 'bus')
+# indicates that a target has a 20% probability of being class 'bike', a 30% probability of being
+# class 'car' etc.
+# It does not make sense to have a true target being a distribution over the possible classes, and
+# therefore the true categorical states will have binary state vectors indicating a specific class
+# (i.e. a '1' at one state vector index, and '0's elsewhere).
+# The :class:`~.CategoricalGroundTruthState` class inherits directly from the base
+# :class:`~.CategoricalState` class.
+#
+# While the category will remain the same, a :class:`~.CategoricalTransitionModel` is used here
+# for the sake of demonstration.
+#
+# The category and timings for one of the ground truth paths will be printed.
+
+from stonesoup.models.transition.categorical import CategoricalTransitionModel
+from stonesoup.types.groundtruth import CategoricalGroundTruthState
+from stonesoup.types.groundtruth import GroundTruthPath
+
+category_transition = CategoricalTransitionModel(transition_matrix=np.eye(3),
+                                                 transition_covariance=0.1 * np.eye(3))
+
```

Comment:
Would a seed be in order?  May have been a conscious omission but just in case you forgot

Reply:
I'd rather keep the example randomized if possible.